airbyte-integrations/connectors/source-file/source_file/client.py

```diff
@@ -248,9 +248,7 @@ def load_nested_json_schema(self, fp) -> dict:
             builder.add_object(json.load(fp))
 
         result = builder.to_schema()
-        if "items" in result and "properties" in result["items"]:
```

Comment:
if we ever had a structure like this, it would fail earlier, on line 248

Reply:
let's add unit test for those case

Reply:
well, I was wrong :( reverted this change
But I also noticed we do not pass required property in the schema. So, now for json and jsonl formats we do not extract properties, but use generated schema as is, only replacing the $schema value. Unit tests added